### DISCUSSION TITLE
Partner zum Einkauf eines WiCAN gesucht

### DISCUSSION THREAD

**Discussion by micha53** [meatpiHQ/wican-fw#253] [General]
Wer hat Interesse, sich gemeinsam mit mir einen WiCAN zu kaufen. Ab 2 Stück entfallen die Versandkosten von 20€. Idealerweise im Raum Erfurt/Thüringen oder Bördekreis.

**Comment by meatpiHQ** [comment]
You can post a message on discord see if anyone is interested.
